wcfsetup/install/files/lib/bootstrap/com.woltlab.wcf.php

```diff
@@ -227,6 +227,11 @@ static function (\wcf\event\endpoint\ControllerCollecting $event) {
             $event->register(new \wcf\system\endpoint\controller\core\notices\DeleteNotice());
             $event->register(new \wcf\system\endpoint\controller\core\notices\GetShowOrder());
             $event->register(new \wcf\system\endpoint\controller\core\notices\ChangeShowOrder());
+            $event->register(new \wcf\system\endpoint\controller\core\reactions\types\Assignable());
+            $event->register(new \wcf\system\endpoint\controller\core\reactions\types\NotAssignable());
```

Comment:
```suggestion
            $event->register(new \wcf\system\endpoint\controller\core\reactions\types\EnableType());
            $event->register(new \wcf\system\endpoint\controller\core\reactions\types\DisableType());
```
I would prefer these names so that it is consistent with other toggle actions.